src/shared/utils/checkRole.ts

```diff
@@ -11,3 +11,10 @@ export const checkIfCanEdit = (roles?: Roles[]) =>
       roles?.includes(Roles.Editor) ||
       roles?.includes(Roles.SuperAdmin),
   );
+
+export const checkIfCanAccessData = (roles?: Roles[]) =>
+  Boolean(
+    isManagerOrOwner(roles?.[0]) ||
+      roles?.includes(Roles.SuperAdmin) ||
+      roles?.includes(Roles.Reviewer),
```

Comment:
There seems to be conflicting information in these tickets. Scenario 4 of [M2-6727](https://mindlogger.atlassian.net/browse/M2-6727) says that “A reviewer can only view or export data.” However, the 3rd case of [M2-6728](https://mindlogger.atlassian.net/browse/M2-6728) says that “Reviewer should not be able to export data in shared account”.

Based on [Cari's comment](https://mindlogger.atlassian.net/browse/M2-6727?focusedCommentId=21467), it seems the reviewer should be able to export data so maybe we should get the description of [M2-6728](https://mindlogger.atlassian.net/browse/M2-6728) updated

Reply:
To potentially further narrow the permissions for a reviewer, another [comment](https://mindlogger.atlassian.net/browse/M2-6727?focusedCommentId=21467) by Cari and an additional reference in [Confluence](https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/172589139/Invite+Reviewer+Checklist) states Reviewer can 
> view or export ONLY assigned user's (Respondents) data

Reply:
@mbanting correct, that's the comment I went with, and Reviewers only see the participants they have assigned + team members so this seems to cover the criteria

Reply:
It's the same comment, just wanted to highlight this so we could clarify for QA testing. I'll resolve this thread now 👍